src/main/java/org/terasology/ligthandshadow/componentsystem/events/FlagDropEvent.java

```diff
@@ -1,32 +1,27 @@
-/*
- * Copyright 2017 MovingBlocks
- *
- * Licensed under the Apache License, Version 2.0 (the "License");
- * you may not use this file except in compliance with the License.
- * You may obtain a copy of the License at
- *
- *      http://www.apache.org/licenses/LICENSE-2.0
- *
- * Unless required by applicable law or agreed to in writing, software
- * distributed under the License is distributed on an "AS IS" BASIS,
- * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
- * See the License for the specific language governing permissions and
- * limitations under the License.
- */
+// Copyright 2021 The Terasology Foundation
+// SPDX-License-Identifier: Apache-2.0
 package org.terasology.ligthandshadow.componentsystem.events;
 
 import org.terasology.engine.entitySystem.entity.EntityRef;
 import org.terasology.engine.entitySystem.event.Event;
-import org.terasology.engine.network.BroadcastEvent;
 
-@BroadcastEvent
+/**
+ * This event causes the player to drop the flag.
+ */
 public class FlagDropEvent implements Event {
```

Comment:
To give this a bit more of a "trigger" characteristic and align it with `GiveFlagEvent`, I'd propose to rename it to `DropFlagEvent`